pkg/apis/noobaa/v1alpha1/noobaa_types.go

```diff
@@ -175,6 +175,9 @@ type NooBaaSpec struct {
 	// +nullable
 	// +optional
 	DisableLoadBalancerService bool `json:"disableLoadBalancerService,omitempty"`
+
+	// +optional
+	AWSSTSRoleARN *string `json:"awsSTSRoleARN,omitempty"`
```

Comment:
didn't we discuss adding the default backing store spec here instead of ARN?
@nb-ohad @nimrod-becker

Reply:
Yes, we did, and I also remember that we all agreed that that will be the way to go. 
@ezio-auditore is this some leftover code? or are we still waiting for some uncommitted work?

Reply:
Yes it is leftover code. I have update the latest commit with the changes